document/src/vespa/document/repo/document_type_repo_factory.cpp

```diff
@@ -22,7 +25,7 @@ EmptyFactoryCheck::~EmptyFactoryCheck()
 {
     if (!DocumentTypeRepoFactory::empty()) {
         std::cerr << "DocumentTypeRepoFactory not empty at shutdown" << std::endl;
-        abort();
+        LOG_ABORT("should not be reached");
```

Comment:
Here we should just assert(!"DocumentTypeRepoFactory not empty at shutdown"); instead of writing to cerr and no abort.